FinSearchUnified/tests/PluginTest.php

```diff
@@ -8,16 +8,24 @@
 
 class PluginTest extends TestCase
 {
+
     protected static $ensureLoadedPlugins = [
-        'FinSearchUnified' => [],
+        'FinSearchUnified' => [
+            'ShopKey' => 'ABCD0815'
+        ],
     ];
 
+
     public function testCanCreateInstance()
     {
         /** @var Plugin $plugin */
         $plugin = Shopware()->Container()->get('kernel')->getPlugins()['FinSearchUnified'];
 
         $this->assertInstanceOf(Plugin::class, $plugin);
+
+        // reset articles data upon starting of test
+        $this->sResetArticles();
```

Comment:
Since this has to be done after each test case, I would suggest executing it in a tear down method.